test/unit_test_import_xml.py

```diff
@@ -12,14 +13,24 @@ def setUp(self):
         self.tool = gxp.import_xml("test/import_xml.xml")
 
 
+class TestOverrides(TestImport):
+    def test_override(self):
+        co = "bash foo.sh > output1"
+        col = co.split(' ')
+        self.tool.command_override = col
+        exml = self.tool.export()
+        self.assertTrue(self.tool.command_override == col)
```

Comment:
```suggestion
        self.assertEqual(self.tool.command_override, col)
```

Reply:
Thanks for fixing - this will work fine for me.